app/controllers/new_admin/users_controller.rb

```diff
@@ -6,6 +6,8 @@ class UsersController < ApplicationController
 
     def show
       @user = User.find(params[:id]).decorate
+      @user_allocations = Allocation.where(user_id: params[:id]).decorate
+      @performance_evaluations = Evaluation.joins(:questionnaire).where(questionnaires: { kind: :performance }, evaluated_id: params[:id]).order(created_at: :desc).decorate
```

Comment:
You could use the scope for the questionnaire here with the merge method https://api.rubyonrails.org/classes/ActiveRecord/SpawnMethods.html#method-i-merge